packages/xo-web/src/common/report-bug-button.js

```diff
@@ -24,32 +29,78 @@ export const reportBug = ({ formatMessage, message, title }) => {
   )
 }
 
-const ReportBugButton = ({
-  formatMessage,
+const SUPPORT_PANEL_URL = `${window.location.origin}/supportPanel/create/ticket`
+
+const reportBugWithFiles = ({
```

Comment:
move it in  'common/xo'?

Reply:
This function not interact with `xo` API.